satellite_passes.py

```diff
@@ -0,0 +1,31 @@
+# %% codecell
+from datetime import datetime, timezone, timedelta
+from skyfield.api import Loader, Topos
+from skyfield.almanac import get_satellite_passes
+
+# %% codecell
+load = Loader("./", verbose=True, expire=True)
+
+# Load timescale
+ts = load.timescale()
+
+# Load planets, get Earth and set the ground station
+ephemeris = load("de421.bsp")
+ground_station = Topos('44 N', '11 E', elevation_m=70)
+
+# Load satellites
+satellites = load.tle("http://celestrak.com/NORAD/elements/weather.txt")
```

Comment:
What if I don't have internet, or what if NOAA18 suddenly gets hit by an asteroid? I love to write code when I'm on an airplane where there are fewer interruptions? There is already a hard coded TLE for the ISS in https://github.com/skyfielders/python-skyfield/blob/master/skyfield/tests/test_earth_satellites.py#L9 - maybe reuse that?